### DISCUSSION TITLE
如何实现右击空白处弹出菜单

### DISCUSSION THREAD

**Discussion by LTKy** [antvis/X6#2547] [Q&A]
如何实现右击空白处弹出菜单

**Comment by kuhnwei** [comment]
使用[鼠标事件](https://x6.antv.vision/zh/docs/tutorial/intermediate/events/#%E9%BC%A0%E6%A0%87%E4%BA%8B%E4%BB%B6) `blank:contextmenu` 事件来实现，右击空白处弹出菜单。

大致流程就是：
1. 空白处点击鼠标右键；
2. 获取鼠标当前坐标；
3. 根据鼠标坐标设置菜单dom元素的坐标以及显示菜单；
4. 在右击事件里添加 监听事件:除了点击菜单之外其它的点击触发菜单的关闭隐藏；
